source/localizable/object-model/computed-properties.md

```diff
@@ -1,12 +1,15 @@
 ## What are Computed Properties?
 
-In a nutshell, computed properties let you declare functions as properties. You create one by defining a computed property as a function, which Ember will automatically call when you ask for the property. You can then use it the same way you would any normal, static property.
+In a nutshell, computed properties let you declare functions as properties.
+You create one by defining a computed property as a function, which Ember will automatically call when you ask for the property.
+You can then use it the same way you would any normal, static property.
 
 It's super handy for taking one or more normal properties and transforming or manipulating their data to create a new value.
 
 ### Computed properties in action
 
-We'll start with a simple example:
+We'll start with a simple example.
+We have a `Person` object with `firstName` and `lastName` properties, but we also want to a `fullName` property that will update whether one of the names changes so it can join them:
```

Comment:
Also `changes so it can join them` could maybe be removed or rewritten in a separate sentence